parachain/pallets/basic-channel/src/outbound/mod.rs

```diff
@@ -65,7 +65,7 @@ pub mod pallet {
 
 		/// Max number of messages per commitment
 		#[pallet::constant]
-		type MaxMessagesPerCommit: Get<u64>;
+		type MaxMessagesPerCommit: Get<u32>;
```

Comment:
☝️ Because BoundedVec's [Decode implementation](https://crates.parity.io/frame_support/storage/bounded_vec/struct.BoundedVec.html#impl-Decode) requires the bound to be a u32.